README.md

```diff
@@ -214,22 +214,6 @@ Setup test config
     # local
     ingres_odbc=ingres:///sa
 
-Code change needed to SA:
-
-    diff --git a/test/requirements.py b/test/requirements.py
-    index cf9168f5a..fcc4f37a0 100644
-    --- a/test/requirements.py
-    +++ b/test/requirements.py
-    @@ -394,6 +394,9 @@ class DefaultRequirements(SuiteRequirements):
-             elif against(config, "oracle"):
-                 default = "READ COMMITTED"
-                 levels.add("AUTOCOMMIT")
-    +        elif against(config, "ingres"):
-    +            default = "READ COMMITTED"
-    +            levels.add("AUTOCOMMIT")  # probably needed, not sure what this is though - assuming tests are not commiting and expecting autocommit semantics
-             else:
-                 raise NotImplementedError()
-
 Run (all) tests:
```

Comment:
Presumably this is no longer needed as changes already present? Information missing from commit. If this is the case, please update the PR description (no need to change history, unless you want to).

Reply:
Re-tested and confirmed that these changes are still required when using the Ingres connector with the unit tests of SQLAlchemy 1.x. The changes are not required with the 1.x dialect compliance suite nor with any of the SQLAlchemy 2.x tests (unit or dialect compliance).

Moved the instructions from README.md to README.testsuite.md.